tasks/set_vars.yml

```diff
@@ -4,7 +4,28 @@
 - name: Lookup applications and set the apps var
   set_fact: apps="{{ lookup('pipe', 'python plugins/utils/lookup_apps.py') | from_yaml }}"
 
-- name: Display looked up applications
+# If the playbook is invoqued with the filter_apps extra var, we restrict apps
```

Comment:
if we follow the same logic as for "customer" and "env_type", it should be called "app_filter" rather then "filter_apps"?. WDYT?

Reply:
Given the name of this branch, I think it makes sense :wink: The past me already agreed.